tests/scripts/all.sh

```diff
@@ -295,13 +300,18 @@ cleanup()
     rm -f programs/test/cmake_package_install/Makefile
     rm -f programs/test/cmake_package_install/cmake_package_install
 
-    if [ -f "$CONFIG_BAK" ]; then
-        mv "$CONFIG_BAK" "$CONFIG_H"
-    fi
+    # Restore files that may have been clobbered by the job
+    for x in $files_to_back_up; do
+        cp -p "$x$backup_suffix" "$x"
+    done
+}
 
-    if [ -f "$CRYPTO_CONFIG_BAK" ]; then
-        mv "$CRYPTO_CONFIG_BAK" "$CRYPTO_CONFIG_H"
-    fi
+final_cleanup () {
+    cleanup
+
+    for x in $files_to_back_up; do
+        rm -f "$x$backup_suffix"
+    done
```

Comment:
Question: should we also invoke `make neat` in this new `final_cleanup`?

Reply:
I don't think so. The files additionally cleaned by `make neat` only depend on the library source, not on any configuration or platform, so it's ok to leave them behind.

Reply:
I agree it's not a problem to leave them behind, but I thought it would be more symmetric to end with `make neat` after we started with `make generated_files`. Anyway that's not important enough to argue about it, so I'm marking this as resolved.